packages/api/tests/utils/ensembleUtils.ts

```diff
@@ -15,16 +15,89 @@
 
 import mocks from '../mock.factories.js';
 import { ENSEMBLE_BRAND } from '../../src/util/ensemble.js';
+import { IrrigationPrescription } from '../../src/util/ensembleService.types.js';
+import TaskModel from '../../src/models/taskModel.js';
+import { AddonPartner, Farm, Location, ManagementPlan } from '../../src/models/types.js';
 
-export interface Farm {
-  farm_id: string;
-}
-
-export const connectFarmToEnsemble = async (farm: Farm) => {
+export const connectFarmToEnsemble = async (farm: Farm, partner?: AddonPartner) => {
   const [farmAddon] = await mocks.farm_addonFactory({
     promisedFarm: Promise.resolve([farm]),
-    promisedPartner: mocks.addon_partnerFactory({ name: ENSEMBLE_BRAND }),
+    promisedPartner: partner
+      ? Promise.resolve([partner])
+      : mocks.addon_partnerFactory({ name: ENSEMBLE_BRAND, id: undefined }),
   });
 
   return { farmAddon };
 };
+
+type fakeIrrigationPrescriptionsProps = {
```

Comment:
This isn't currently like our other "fakes" I would like to fix that but maybe once it is not doing both beta and tests mocking.